### ISSUE TITLE
[FEATURE REQUEST] Ability to execute commands after a player loses.

### ISSUE THREAD

**Issue by DuckMC** [goodbyepavlyi/LavaRise#4]
### [FEATURE REQUEST] Ability to execute commands after a player loses. 

**Description:** Add the ability to run specific commands when a player loses. 

**Additional note:** Currently there is the ability to run commands when the player wins the game. It would be a good enhancement if that option is also there but for players who lose a game.

**Comment by goodbyepavlyi** [comment]
Thank you for the suggestion! This feature will be included in the upcoming v2.0.1 release. 
Commit: (93bfa8ac11f27e0775e17947d2f05b9eda183983)

**⚠ BREAKING CHANGE**: The replacement value in `game.commands.winner` is now `%player%` instead of `%winner%`.
```diff
game:
  commands:
    winner:
-      - tell %winner% You won the game!
+      - tell %player% You won the game!
```

**Comment by DuckMC** [comment]
Thanks for implementing the requested functionality! I look forward to seeing this feature soon in the upcoming v2.0.1 release.
